src/types/index.ts

```diff
@@ -104,6 +104,8 @@ export enum ENTITY {
     COLLECTION = 'collection',
 }
 
+export type ENTITY_WITH_CREATE = ENTITY.CAPTURE | ENTITY.MATERIALIZATION;
```

Comment:
Type names are traditionally written in pascal case. Is there any reason to diverge from that convention here?

Additionally, the type name gives me a _cause for pause_ as it is slightly ambiguous. If someone new to the codebase were to come across this type, it may be difficult to understand its role. How does `EntityWithCreateWorkflow` sound to you? A seemingly minimal change, but a reasonable increase in context.

**NOTE:** At the very least, I would like to see the `ENTITY` enumeration name converted to pascal case in the near future. The convention requiring names of enumerations to be plural and types to be written in a singular form of a noun is one of which I am particularly fond. While it can be broken on rare occasion, it eases the identification of types and enumerations.

Reply:
I'll update the name.

I think worrying about naming things different between type and enum is not needed. Folks can easily find the definition in an IDE. Also, I don't want any "rules" on naming and stuff that is not declared in a Prettier or ESLint rule. So if you want that to be a rule then you are free to see if you can get one added to the Prettier/Linting.